appveyor.yml

```diff
@@ -19,7 +21,9 @@ build_script:
 - cmd: >-
     IF "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2015" (SET COMPILER=vs2015)
 
-    IF "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2017" (SET COMPILER=vs2017)
+    IF "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2017" (SET COMPILER=vs2017 && SET ARM64_SUPPORTED=1)
+
+    IF "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2019" (SET COMPILER=vs2019 && SET ARM64_SUPPORTED=1)
```

Comment:
Missing arm64 dependency install like rtm/sjson-cpp? The appveyor arm64 build fails due to missing things so that's probably it.
Also notice how unlike RTM, ACL has the build matrix split per configuration as well and launches 1 job for debug and one for release per compiler version to avoid timing out.

Reply:
I did notice that difference - so why doesn't RTM use the matrix for configurations? Is it to avoid running -nosimd for debug builds?

Reply:
ACL unit tests are slower because we also test the packing/unpacking math functions which takes a little while. ACL simply started timing out first and I haven't gotten around to or had the need to do the same with RTM yet but it looks like we are there now.